tools/cloud-build/daily-tests/ansible_playbooks/tasks/rescue_ghpc_failure.yml

```diff
@@ -30,11 +30,12 @@
     - delete
     - "{{ deployment_name }}"
 
-- name: Tear Down Cluster
-  changed_when: true  # assume something destroyed
-  run_once: true
```

Comment:
I took this code from another test, so I'm assuming it's functional (besides having tested it).  It seems to now be able to track ghpc so we can get rid of the assumption of something being changed.  Additionally the `run_once` is redundant (at least in the case) because it's delegated to localhost in `slurm-integration-test.yml`.  That being said I've re-added it in just to make sure if any other script uses it, it'll be handled correctly.

Reply:
SG. If we think it is unused then cleaning it up is not a bad idea, but... I would do it in a separate PR to isolate the change and make it easy to identify and roll back if necessary. Your choice if you want to pursue.